docs/index.rst

```diff
@@ -23,10 +23,16 @@ Welcome to GlobalFoundries 0.18UM 3.3V/(5V)6V MCU PDK's documentation!
     :alt: GlobalFoundries PDK Logo Image.
 
 
-Current Status -- |current-status|
-==================================
+.. include:: status.rst
+
+
+Resources
```

Comment:
I like this new section, but I think it **should** be in a separate pull request and not the one fixing warnings.

Reply:
done. removed from new version.